lusidtools/apps/upsert_holdings.py

```diff
@@ -11,19 +12,26 @@
     validate_mapping_file_structure,
     load_json_file,
     cocoon_printer,
+    strip_whitespace,
 )
+import pdb
```

Comment:
Perhaps move this to line 4?
I usually keep the `import` general packages together and then group the LUSID ones separately.

Reply:
that import is redundant, I'll remove it